lightning/src/chain/channelmonitor.rs

```diff
@@ -2857,6 +2870,10 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		&self.funding_info
 	}
 
+	pub fn get_channel_id(&self) -> Option<ChannelId> {
+		self.channel_id
+	}
```

Comment:
This could be a good place to include the default-v1-from-funding-txo logic

Reply:
Yeah agreed. Going to do so during cleanup of the PR